duckduckgo/Android
# ADS: Fix Divider Lint

Task/Issue URL:  https://app.asana.com/0/inbox/1157893581871899/1203130286824893/1203130286824895

### Description
Wrong assertion in DividerTest
I also removed the AlertDialogTest since it's not working. I'll get it fixed and back once we migrate all dialogs

### Steps to test this PR
Build should be green

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203130286824893
